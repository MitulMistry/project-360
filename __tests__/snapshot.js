import { render } from "@testing-library/react";
import Page from "../app/page";

it("renders splash page unchanged", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
