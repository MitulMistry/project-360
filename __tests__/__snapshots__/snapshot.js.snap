// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders splash page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <div
        class="splash"
        data-testid="splash"
      >
        <img
          alt="Graphic logo"
          class="logo"
          data-nimg="1"
          decoding="async"
          height="39"
          loading="lazy"
          src="/graphics/logo.svg"
          style="color: transparent;"
          width="208"
        />
        <img
          alt="Graphic receipt"
          class="graphic"
          data-nimg="1"
          decoding="async"
          height="109"
          loading="lazy"
          src="/graphics/receipt.svg"
          style="color: transparent;"
          width="84"
        />
        <h3
          class="header"
        >
          Project Management Made Simple
        </h3>
        <p
          class="text"
          data-testid="description"
        >
          Create projects, assign tasks, and manage the workflow for your organization.
        </p>
        <a
          class="btnLink"
          data-testid="signup"
          href="/signup"
        >
          <button
            class="button md primary default"
            data-rac=""
            type="button"
          >
            Create Account
          </button>
        </a>
        <a
          data-testid="login"
          href="/login"
        >
          Sign In
        </a>
      </div>
    </main>
  </div>
</div>
`;
